.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
at the end when the PR is finished remove this file.

Reply:
Will do. Might be good to add this path to gitignore

Reply:
Done